_includes/ios/objects.md

```diff
@@ -101,18 +101,20 @@ let playerName = gameScore["playerName"] as String
 let cheatMode = gameScore["cheatMode"] as Bool
 </code></pre>
 
-The three special values are provided as properties:
+The four special values are provided as properties:
 
 <pre><code class="objectivec">
 NSString *objectId = gameScore.objectId;
 NSDate *updatedAt = gameScore.updatedAt;
 NSDate *createdAt = gameScore.createdAt;
+PFACL *ACL = gameScore.acl;
 </code></pre>
 
 <pre><code class="swift">
 let objectId = gameScore.objectId
 let updatedAt = gameScore.updatedAt
 let createdAt = gameScore.createdAt
+let acl = gameScore.acl
```

Comment:
I would make this **ACL**.

Not too familiar with iOS side of things yet but in other locations of iOS docs, **ACL** is capitalized. There are no lowercase spellings of it.

Same for ObjC side of things too.